drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -764,7 +770,10 @@ def _get_item_raw_stats(self):
         placed_items = set(item_id for item_id in all_items if item_id in item_state)
         correct_items = set(item_id for item_id in placed_items if item_state[item_id]['correct'])
 
-        return required_items, placed_items, correct_items
+        decoy_items = set(item_id for item_id in all_items if self._get_item_zones(int(item_id)) == [])
+        decoy_items_unplaced = set(item for item in decoy_items if item not in item_state)
+
+        return required_items, placed_items, correct_items, decoy_items, decoy_items_unplaced
```

Comment:
@arbrandes it might make sense to convert this 5-tuple to a namedtuple, to avoid unpacking and discarding some values ( `required_ids, placed_ids, correct_ids, __, __`)